Dockerfile

```diff
@@ -1,29 +1,37 @@
 FROM python:3.9
 
-RUN SNIPPET="export PROMPT_COMMAND='history -a' && export HISTFILE=/commandhistory/.bash_history" && echo $SNIPPET >> "/root/.bashrc"
+ARG USERNAME=kevin
```

Comment:
I don't mind hardcoding the username at all (that's what we currently have with `root` in effect). Could we change it to something generic like `dev` or similar?

Reply:
Made this modification.  Good idea on dev.  I like that